src/js/utils/specs/date-utils-spec.js

```diff
@@ -0,0 +1,145 @@
+define(
+    [
+        '../../utils/spec-helpers/spec-helpers.js',
+        '../date-utils.js',
+        'moment',
+        'moment-timezone'
+    ],
+    function(
+        SpecHelpers,
+        DateUtils,
+        moment,
+        momentTZ
+    ) {
+        'use strict';
+
+        describe('DateUtils', function() {
+            beforeEach(function() {
+                setFixtures('<div class="test"></div>');
+            });
+
+            describe('stringToMoment', function() {
+                it('returns an empty string if provided no time', function() {
+                    expect(DateUtils.stringToMoment()).toEqual('');
+                });
+                it('will convert a string to an moment.js object', function() {
```

Comment:
Nittiest of the nit: for consistency with the rest of the tests, phrase this in the present tense: "will convert" -> "converts"